src/components/Form/RangeSlider.js

```diff
@@ -70,7 +91,11 @@ RangeSlider.propTypes = {
     /** Define the appearance of the form element. Accepted values are either "inline" or "block" */
     appearance: PropTypes.oneOf(["inline", "block"]),
     /** Becomes a controlled component if onChange function is given */
-    onChange: PropTypes.func
+    onChange: PropTypes.func,
+    /** Custom Style changes of slider track */
+    inputStyle: PropTypes.object,
+    /** Show bubble with values */
+    bubble: PropTypes.bool
```

Comment:
please rename to `showBubble` so that the variable name indicates that this is a boolean value used to decide whether to show the bubble or not